src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -688,7 +688,14 @@ internal async Task<JToken> RuntimeGetPropertiesInternal(SessionId id, DotnetObj
                     case "array":
                         return await context.SdbAgent.GetArrayValues(int.Parse(objectId.Value), token);
                     case "object":
+                    {
+                        if (objectId.SubScheme == "methodId") //getter
+                        {
+                            var objRet = await context.SdbAgent.InvokeMethodInObject(int.Parse(objectId.Value), int.Parse(objectId.SubValue), "", token);
```

Comment:
What about moving the call: *InvokeMethodInObject()* into https://github.com/dotnet/runtime/blob/09bb80e003b09ed5f451e2029ea0bb29e65c42a0/src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs#L1601? This way we will avoid double clicking (in Console App the *subScheme* values are unfolded directly by a single call on *getProperties*, no need for double call). 
I tried to move the function there but so far I didn't get the right result - it seems as if I was getting into infinite loop or something. Maybe that's the reason why someone used this condition on *isAutoExpandable* but if fixing it was possible then it would be prefect.

Reply:
The reason for infinite loop in objects: we are calling GetObjectValues => CreateJObjectForVariableValue => CreateJObjectForObject => GetValueFromDebuggerDisplayAttribute => GetObjectValues. Can we avoid it somehow?